app/models/transcription_field.rb

```diff
@@ -1,20 +1,24 @@
+# frozen_string_literal: true
+
 class TranscriptionField < ApplicationRecord
   belongs_to :collection, optional: true
-  acts_as_list :scope => :collection
+
+  acts_as_list scope: :collection
+
   has_many :table_cells
-  has_many :spreadsheet_columns, -> { order 'position' }, :dependent => :destroy
-  validates :options, presence: true, if: Proc.new {|field| field.input_type == 'select'}, on: [:create, :update]
+  has_many :spreadsheet_columns, -> { order 'position' }, dependent: :destroy
 
+  validates :options, presence: true, if: Proc.new { |field| field.input_type == 'select' }, on: [:create, :update]
   validates :percentage, numericality: { allow_nil: true, greater_than: 0, less_than_or_equal_to: 100 }
   validates :page_number, numericality: { allow_nil: true, greater_than: 0, less_than_or_equal_to: 1000 }
-
+  validates :label, format: { without: /[\[\]\{\}]/, message: "cannot contain '{', '}', '[' or ']'" }
```

Comment:
Validation for checking characters that might be interpreted as hash like '{}' or '[]'

Reply:
Let's remove the `{}` validation -- only the hash element accessor `[]` is used by Rails to create structured parameters, `{}` isn't a problem.  (I just tested that on `development` and `{}` works.